llm/dbrx/README.md

```diff
@@ -98,26 +100,61 @@ HF_TOKEN=xxx sky launch dbrx.yaml -c dbrx --env HF_TOKEN
 
 ```console
 ...
-I 03-27 21:08:53 optimizer.py:690] == Optimizer ==
-I 03-27 21:08:53 optimizer.py:701] Target: minimizing cost
-I 03-27 21:08:53 optimizer.py:713] Estimated cost: $4.1 / hour
-I 03-27 21:08:53 optimizer.py:713]
-I 03-27 21:08:53 optimizer.py:836] Considered resources (1 node):
-I 03-27 21:08:53 optimizer.py:906] -----------------------------------------------------------------------------------------------------
-I 03-27 21:08:53 optimizer.py:906]  CLOUD   INSTANCE               vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN
-I 03-27 21:08:53 optimizer.py:906] -----------------------------------------------------------------------------------------------------
-I 03-27 21:08:53 optimizer.py:906]  AWS     p4d.24xlarge[Spot]     96      1152      A100:8         us-east-2b      4.13          ✔
-I 03-27 21:08:53 optimizer.py:906]  GCP     a2-ultragpu-4g[Spot]   48      680       A100-80GB:4    us-east4-c      7.39
-I 03-27 21:08:53 optimizer.py:906]  GCP     a2-highgpu-8g[Spot]    96      680       A100:8         us-central1-a   11.75
-I 03-27 21:08:53 optimizer.py:906]  GCP     a2-ultragpu-8g[Spot]   96      1360      A100-80GB:8    us-east4-c      14.79
-I 03-27 21:08:53 optimizer.py:906]  GCP     a2-megagpu-16g[Spot]   96      1360      A100:16        us-central1-a   22.30
-I 03-27 21:08:53 optimizer.py:906] -----------------------------------------------------------------------------------------------------
+I 03-28 08:40:47 optimizer.py:690] == Optimizer ==
```

Comment:
May need to update the price in parens above, or remove that parens?

Reply:
good catch - removing pricing since it's dependent on clouds enabled/current prices. Readers can still see price in the optimizer table by expanding it.